Makefile

```diff
@@ -122,12 +122,18 @@ $(LOCALBIN):
 
 GOLANGCI_LINT_VERSION ?= v1.61.0
 GOLANGCI_LINT = $(LOCALBIN)/golangci-lint-$(GOLANGCI_LINT_VERSION)
+GOCHECKNOGLOBALS := 4d63.com/gochecknoglobals
 
 .PHONY: golangci-lint
 golangci-lint: $(GOLANGCI_LINT) ## Download golangci-lint locally if necessary.
 $(GOLANGCI_LINT): $(LOCALBIN)
 	$(call go-install-tool,$(GOLANGCI_LINT),github.com/golangci/golangci-lint/cmd/golangci-lint,${GOLANGCI_LINT_VERSION})
 
+.PHONY: gochecknoglobals
+gochecknoglobals: $(GOCHECKNOGLOBALS) ## Download gochecknoglobals locally if necessary.
+$(GOCHECKNOGLOBALS): $(LOCALBIN)
+    $(call go-install-tool,$(GOCHECKNOGLOBALS),latest)
```

Comment:
Why we need to download this? Don't we already reuse the `gochecknoglobals` linter which is part of `golangci-lint`?

Reply:
Yes, we don't need to anymore, thanks for the heads up.